java/com/google/location/lbs/activity/audioset/dataprocessing/incomplete-example.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+option java_outer_classname = "IncompleteExample";
+
+message Example {
+    string ytid = 1;
+    int32 start = 2;
+    int32 end = 3;
+
+    message Label {
```

Comment:
you plan to have more fields in Label?

Reply:
No. I did not plan to have more fields in Label.